Backend/Remora.Discord.Unstable/Unstable/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -40,16 +43,29 @@ public static class ServiceCollectionExtensions
         /// Adds experimental features from the Discord API.
         /// </summary>
         /// <param name="serviceCollection">The service collection.</param>
+        /// <param name="optionsName">The name of the serializer options, if any. You should probably leave this set to the default value.</param>
         /// <returns>The service collection, with the services.</returns>
         public static IServiceCollection AddExperimentalDiscordApi
         (
-            this IServiceCollection serviceCollection
+            this IServiceCollection serviceCollection,
+            string? optionsName = "Discord"
         )
         {
-            serviceCollection.Configure<JsonSerializerOptions>(jsonOptions =>
+            serviceCollection.Configure<JsonSerializerOptions>(optionsName, jsonOptions =>
             {
+                jsonOptions.Converters.Add(new UnstableVoicePayloadConverter());
```

Comment:
You can use the extension method `AddConverter` for this - it'll insert it before all other converters in the list, ensuring it takes priority over existing ones.